jdaviz/app.py

```diff
@@ -1179,7 +1179,7 @@ def resize(stack_items):
         # resize tray items
         for tray_item in self.state.tray_items:
             # access the actual plugin object (there is no store for plugins)
-            tray_obj = self.widgets.get(tray_item['widget'].split('IPY_MODEL_')[1])
+            tray_obj = widget_serialization['from_json'](tray_item['widget'], None)
```

Comment:
Can't we create our down `widgets` property here to return either the old or new API based on ipywidgets version?

Reply:
Or maybe a new method name altogether or something?

Reply:
I would have expected my previous commit to work if that would work.  But please feel free to try (either a separate PR or push here) as I agree that is cleaner than this!

Reply:
Mario said your solution is very nice though, so maybe you can ignore my comments.